src/sentry/performance_issues/detectors/query_injection_detector.py

```diff
@@ -19,6 +20,25 @@
 MAX_EVIDENCE_VALUE_LENGTH = 10_000
 
 
+def disable_query_injection_detection_for_project(project: Project) -> None:
```

Comment:
might be useful to generalize this in case we want to default disable detection for other detectors (which is currently the plan). maybe we can pass in the setting value so that we can just call the function with a diff input if we want to disable one of the HTTP detectors for example.